devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -119,7 +119,7 @@
 
     private static final String QUARKUS_PLUGIN_GROUPID = "io.quarkus";
     private static final String QUARKUS_PLUGIN_ARTIFACTID = "quarkus-maven-plugin";
-    private static final String QUARKUS_PREPARE_GOAL = "prepare";
+    private static final String QUARKUS_PREPARE_GOAL = "generate-code";
```

Comment:
The constant should probably be renamed too?

Reply:
done